lib/miq_automation_engine/service_models/mixins/miq_ae_service_miq_request_mixin.rb

```diff
@@ -17,6 +17,7 @@ def user_message=(msg)
 
   def set_option(key, value)
     ar_method do
+      reload
```

Comment:
@bzwei To protect this operation @Fryguy made the suggestion yesterday that we add a lock on the object here.

Reply:
@Fryguy Should this method be in the ServiceModel or in the app/models? Maybe a new method that uses a transaction block. I am not sure if we have added locks or transaction blocks in the ServiceModel.

Reply:
My opinion is that this lock/reload/save block is only being added to address an automate method issue so there is no problem with the changes living here for now.